exonum_client/proofs/list_proof/key.py

```diff
@@ -1,6 +1,7 @@
 """ ProofListKey Module. """
 from typing import Dict, Any
 from functools import total_ordering
+import logging
```

Comment:
Regarding the whole proof module: everything related to the verification of the proof (e.g. malformed proof etc) failure is not a fault of the library, thus it's not `critical` and not `error`: it's just user received malformed proof. It can be `warning`, but not more than that.

And again: if there is a possibility to provide helpful info about the context of the error, please do so.